main.py

```diff
@@ -1,9 +1,17 @@
-class Animal:
+class Pet:
     def __init__(self, attack:int, health:int) -> None:
         self.attack = attack
         self.health = health
     
-    def attack(self, other:"Animal") -> None:
+    def attack(self, other:"Pet") -> None:
         other.health -= self.attack
         if other.health < 0:
             other.health = 0
+
+class Army:
+    def __init__(self) -> None:
+        self.army = []
+
+    def add_pet(self, pet:Pet):
+        if len(self.army) < 5:
+            self.army.append(pet)
```

Comment:
should raise error if not
```suggestion
            self.army.append(pet)
    else:
             raise ValueError
```